src/actions/deleteCategoryActions.js

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+
+import {
+  DELETE_CATEGORY_REQUEST,
+  DELETE_CATEGORY_SUCCESS,
+  DELETE_CATEGORY_FAILURE,
+} from '../types';
+import config from '../../config';
+
+/**
+ * @function deleteCategoryRequest
+ * @return {Object} {{type: DELETE_CATEGORY_REQUEST}}
+ */
+export const deleteCategoryRequest = () => (
+  {
+    type: DELETE_CATEGORY_REQUEST,
+  }
+);
+
+  /**
+   * delete Category failure action creator
+   * @function deleteCategoryFailure
+   * @param {Object} error - Error object with error information
+   * @return {Object} {{type: DELETE_CATEGORY_FAILURE, error: error}}
+   */
+export const deleteCategoryFailure = error => (
+  {
+    type: DELETE_CATEGORY_FAILURE,
+    error,
+  }
+);
+
+  /**
+   * @function deleteCategorySuccess
+   * @param {Array} category - object with info on deleted category
+   * @return {Object} {{type: DELETE_CATEGORY_SUCCESS, bool: bool}}
+   */
+export const deleteCategorySuccess = id => (
```

Comment:
Please verify with `bend` that when an `activity type/ category` is deleted only its id is returned

Reply:
I believe the delete protocol's convention says a `204` is returned with no payload.
The id I provide isn't from the response to the request but rather the exact id passed in with the request. See the *.then callback dispatch* in the thunk below and advise from there on

Reply:
👍